src/utils/ddb/tests/SConscript

```diff
@@ -26,16 +26,16 @@ def scons():
     libs = ['vos', 'daos_common_pmem', 'abt', 'gurt', 'uuid', 'bio', 'cart',
             'cmocka', 'ddb']
     # spdk libraries
-    libs += ['spdk_event', 'spdk_log']
-    libs += ['spdk_bdev', 'spdk_blob', 'spdk_blob_bdev', 'spdk_json']
-    libs += ['spdk_nvme', 'spdk_init', 'spdk_thread', 'spdk_log']
-    libs += ['spdk_env_dpdk', 'spdk_thread', 'spdk_bdev', 'rte_mempool']
-    libs += ['rte_mempool_ring', 'rte_bus_pci', 'rte_pci', 'rte_ring']
-    libs += ['rte_mbuf', 'rte_eal', 'rte_kvargs', 'spdk_bdev_aio']
-    libs += ['spdk_bdev_nvme', 'spdk_blob', 'spdk_nvme', 'spdk_util']
-    libs += ['spdk_json', 'spdk_jsonrpc', 'spdk_rpc', 'spdk_trace']
-    libs += ['spdk_sock', 'spdk_log', 'spdk_notify', 'spdk_blob_bdev']
-    libs += ['spdk_vmd', 'spdk_event_bdev', 'spdk_init', 'rte_power']
+    spdk_libs = ['spdk_event', 'spdk_log']
```

Comment:
Why is this not one statement such as here?  So long as you're changing it, that would be more efficient

https://github.com/daos-stack/daos/blob/d38cd11a1fcfa175ce73a881051f20a04efae0cf/src/SConscript#L5-L10

Reply:
IMHO these lists are so long it doesn't look right no matter what. xD Done.